src/haddock/modules/analysis/clustfcc/defaults.yaml

```diff
@@ -1,51 +1,51 @@
 executable:
   default: src/contact_fcc
   type: file
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
-  explevel: easy
+  title: Relative path to contact_fcc executable.
+  short: Relative path to contact_fcc executable.
+  long: As contact_fcc is an external tool, its path must be provided. Basically, this parameter should not be changed, as the tool is downloaded at the same time as haddock3. Nevertheless, one could try to have the contact_fcc tool written in another programming language or whatever.
+  group: executable
+  explevel: expert
 contact_distance_cutoff:
   default: 5.0
   type: float
-  min: -9999
-  max: 9999
+  min: 0.000
+  max: 999.999
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Cutoff distance (A) for contact detection.
+  short: Cutoff distance (A) for contact detection. By default 5.0 Angstroms.
+  long: Cutoff distance between two atoms for contact detection. By default 5.0 Angstroms.
+  group: analysis
   explevel: easy
 fraction_cutoff:
-  default: 0.6
+  default: 0.600
   type: float
-  min: -9999
-  max: 9999
+  min: 0.000
+  max: 1.000
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Fraction of common contacts to not be considered a singleton model.
```

Comment:
singleton model is not a very common term and will be confusing to users. Why not a simpler description, e.g. minimum fraction of common contacts to be considered in a cluster

Reply:
And for the other levels, I would write it as minimum fraction to be considered in a cluster (rather than not be considered - we want to cluster at the end :-) )